README.md

```diff
@@ -228,7 +228,7 @@ Maintainers ([@open-telemetry/js-maintainers](https://github.com/orgs/open-telem
 | Package                                         | Description                                                                                                                                                                                                                                           |
 | ----------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [@opentelemetry/sdk-trace-base][otel-tracing]   | This module provides a full control over instrumentation and span creation. It doesn't load [`async_hooks`](https://nodejs.org/api/async_hooks.html) or any instrumentation by default. It is intended for use both on the server and in the browser. |
-| [@opentelemetry/sdk-metrics-base][otel-metrics] | This module provides instruments and meters for reporting of time series data.                                                                                                                                                                        |
+| [@opentelemetry/sdk-metrics][otel-metrics] | This module provides instruments and meters for reporting of time series data.                                                                                                                                                                        |
```

Comment:
The link "otel-metrics" needs to be updated too.

Reply:
I didn't update that one because I haven't moved the code yet, so the link will be broken, will update in following PR changing the actual metrics sdk code path

Reply:
Why does this PR not change the directory path?